sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/java/com/azure/data/schemaregistry/avro/SchemaRegistryAvroDeserializer.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.schemaregistry.avro;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.data.schemaregistry.AbstractDataDeserializer;
+import com.azure.data.schemaregistry.SerializationException;
+import com.azure.data.schemaregistry.client.CachedSchemaRegistryClientBuilder;
+
+/**
+ * A deserializer implementation capable of automatedly deserializing encoded byte array payloads into Java objects by
+ * fetching payload-specified schemas from the Azure Schema Registry store.
+ *
+ * SchemaRegistryAvroDeserializer instances should be built using the static Builder class.
+ *
+ * Pluggable with the core Azure SDK Deserializer interface.
```

Comment:
Need to add `<p>` tags to have Javadocs generate with line separators.

```suggestion
 * A deserializer implementation capable of automatedly deserializing encoded byte array payloads into Java objects by
 * fetching payload-specified schemas from the Azure Schema Registry store.
 * <p>
 * SchemaRegistryAvroDeserializer instances should be built using the static Builder class.
 * <p>
 * Pluggable with the core Azure SDK Deserializer interface.
```